backend/battle/models.py

```diff
@@ -24,7 +24,7 @@ class Team(models.Model):
 class PokemonTeam(models.Model):
     team = models.ForeignKey(Team, on_delete=models.CASCADE, related_name="teams")
     pokemon = models.ForeignKey(Pokemon, on_delete=models.CASCADE, related_name="+")
-    order = models.PositiveIntegerField()
+    order = models.PositiveIntegerField(null=True)
```

Comment:
Sorry if you already discussed this with Rebeca, but does this really need to be `null=True`?

Reply:
Previously it was a requirement because I used to create a register on PokemonTeam without order and then I updated it. But now it's not like this, so I'm going to change it. Thank you one more time :)